spec/controllers/sessions_controller_spec.rb

```diff
@@ -63,4 +63,28 @@
       end
     end
   end
+
+  describe 'DELETE#destroy' do
+    it 'renders a redirect response' do
```

Comment:
it's not `render`

Reply:
Fixed in d19c7b8eb9d2313efb53ac48aa38270a0da94aa8